html5-boilerplate/new-site/index.html

```diff
@@ -82,48 +87,58 @@ <h2>React</h2>
         <a href="https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started">Learn more</a>
         </div>
       </div>
-    </div>
+    
   
-    <div class="card-4">
-      <img src="./img/react hooks.png"/>
+    <div class="card">
+      <div class="image"><img src="./img/react hooks.png"/></div>
```

Comment:
Be careful of spaces in file names. In general when coding we avoid spaces and stick to `-` or `_` to separate words. Or camel case in other instances. So here you should rename the file to `react-hooks.png`.  This is because of the way that files are resolved and in general how the operating system receives text-based commands.

Reply:
I have made changes and removed space from the file name.